neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/embedded/impl/EmbeddedNeo4jSchemaDefiner.java

```diff
@@ -64,42 +59,6 @@ protected void createUniqueConstraintsIfMissing(DatastoreProvider provider, List
 		EmbeddedNeo4jDatastoreProvider neo4jProvider = (EmbeddedNeo4jDatastoreProvider) provider;
 		GraphDatabaseService neo4jDb = neo4jProvider.getDatabase();
 		Transaction tx = neo4jDb.beginTx();
-		if ( tx instanceof TopLevelTransaction ) {
-			TopLevelTransaction topLevelTransaction = (TopLevelTransaction) tx;
-			log.infof( "TopLevelTransaction topLevelTransaction : %s", topLevelTransaction.toString() );
-			try {
-				Field privateTransactionField = TopLevelTransaction.class.getDeclaredField( "transaction" );
-				privateTransactionField.setAccessible( true );
-				KernelTransactionImplementation kernelTransaction = (KernelTransactionImplementation) privateTransactionField.get( topLevelTransaction );
-				log.infof( "kernelTransaction : %s", kernelTransaction.getTransactionId() );
-			}
-			catch (IllegalStateException ise) {
-				log.warn( "New toplevel transaction created!" );
-			}
-			catch (Exception e) {
-				log.error( "ERROR!", e );
-			}
-		}
-		else {
-			PlaceboTransaction placeboTransaction = (PlaceboTransaction) tx;
-			log.infof( "PlaceboTransaction placeboTransaction : %s", placeboTransaction );
-
-			try {
-				Field privateTransactionField = PlaceboTransaction.class.getDeclaredField( "currentTransaction" );
-				privateTransactionField.setAccessible( true );
-				KernelTransactionImplementation kernelTransaction = (KernelTransactionImplementation) privateTransactionField.get( placeboTransaction );
-
-				log.infof( "kernelTransaction : %s", kernelTransaction.getTransactionId() );
-			}
-			catch (IllegalStateException ise) {
-				log.warn( "New transction created!" );
-			}
-			catch (Exception e) {
-				log.error( "ERROR!", e );
-			}
-		}
-
-
```

Comment:
Are you telling me that all this code was to just log something? Maybe next time, at least, put it in a separate method so that the purpose is clear, please.